digital-twin/device/src/main/java/com/microsoft/azure/sdk/iot/digitaltwin/device/AbstractDigitalTwinComponent.java

```diff
@@ -59,17 +59,27 @@ protected AbstractDigitalTwinComponent(@NonNull String digitalTwinComponentName,
      * The application may invoke this function as many times as it wants, subject to the device's resource availability. The application does NOT need to wait for each send to finish.
      * The SDK will automatically attempt to retry the telemetry send on transient errors.
      *
+     * @param payload DigitalTwin telemetry message payload. It should be a valid json string represent a map. The key is property name. the JSON should look like:
```

Comment:
grammar: capital T in "the" from "the JSON should look like:"

Reply:
fixed